adapters/Pangle/Public/Headers/GADMediationAdapterPangle.h

```diff
@@ -20,6 +20,8 @@ typedef NS_ENUM(NSInteger, GADPangleErrorCode) {
     GADPangleErrorInvalidServerParameters = 101,
     /// Banner size mismatch.
     GADPangleErrorBannerSizeMismatch = 102,
+    /// Init ad error
+    GADPangleErrorInitAd = 103,
```

Comment:
If Pangle SDK returns an error if the ad slot fails to initialize, we can skip that check in the adapter, and remove this error code.

However if it doesn't, update this to:
```
/// Pangle ad slot fails to initialize.
GADPangleErrorAdSlotInitializationFailure = 103,
```